src/assets/TokenListController.ts

```diff
@@ -317,19 +251,28 @@ export class TokenListController extends BaseController<
           (token) => !duplicateSymbols.includes(token.symbol),
         );
         for (const token of uniqueTokenList) {
-          tokenList[token.address] = token;
+          const formattedToken: RawToken = {
+            ...token,
+            aggregators: formatAggregatorNames(
+              (token.aggregators as AggregatorKey[]) || [],
```

Comment:
Is it possible to not typecast this? How do you know that this contains `AggregatorKey`s?